clang/lib/AST/ExprConstant.cpp

```diff
@@ -8669,6 +8669,25 @@ class PointerExprEvaluator
     return true;
   }
 
+  bool VisitUniqueStableNameExpr(const UniqueStableNameExpr *E) {
+    std::string ResultStr = E->ComputeName(Info.Ctx);
+
+    Info.Ctx.UniqueStableNameEvaluatedValues[E] = ResultStr;
+
+    QualType CharTy = Info.Ctx.CharTy.withConst();
+    APInt Size(Info.Ctx.getTypeSize(Info.Ctx.getSizeType()), ResultStr.size());
```

Comment:
This gets you to the end of the string but not to the null terminator, I think we need a `+ 1` in here for that case?

Reply:
Yep, you're right!  Good catch.  Patch in coming.